quests.md

```diff
@@ -0,0 +1,272 @@
+|---------------|--------|-------------------------------------|
+| longitude     | float  |                                     |
+| latitude      | float  |                                     |
+| pokestop_url  | string |                                     |
+| pokestop_name | string |                                     |
+| updated       | int    | epoch time                          |
+| conditions    | array  | See [conditions](#conditions) table |
+| rewards       | array  | See [rewards](#rewards) table       |
+
+```json
+[
+  {
+    "type": "quest",
+    "message": {
+      "longitude": float,
+      "latitude": float,
+      "pokestop_url": string,
+      "pokestop_name": string,
+      "updated": epoch int,
+      "conditions": [...],
+      "rewards": [...]
+    }
+  }
+]
+```
+
+## Conditions
+This spec will cover most of the requirements, but when more information is needed, the detail field should be included
+
+Otherwise, the detail field will not be checked
+
+| Key           | Type   | Description                                                                   |
+|---------------|--------|-------------------------------------------------------------------------------|
+| type          | int    | See [Conditions Type Array](#conditions-type-array)                           |
+| amount        | int    | A generic number representing how many times something must be done or aquired|
+| detail        | object | See [Conditions Detail Sections](#conditions-detail) for further detail       |
+
+```json
+[
+    {
+      "type": int,
+      "amount": int,
+      "detail": {...}
+    }
+]
+```
+
+### Conditions Type Array
+This came directly out of the protos, and the `type` int field for every condition should represent one of these
+
+Keep in mind that all values will be accepted, but not all will be valid, even values in this list
+```
+enum ConditionType {
+    UNSET = 0;
+    WITH_POKEMON_TYPE = 1;
+    WITH_POKEMON_CATEGORY = 2;
+    WITH_WEATHER_BOOST = 3;
+    WITH_DAILY_CAPTURE_BONUS = 4;
+    WITH_DAILY_SPIN_BONUS = 5;
+    WITH_WIN_RAID_STATUS = 6;
+    WITH_RAID_LEVEL = 7;
+    WITH_THROW_TYPE = 8;
+    WITH_WIN_GYM_BATTLE_STATUS = 9;
+    WITH_SUPER_EFFECTIVE_CHARGE = 10;
+    WITH_ITEM = 11;
+    WITH_UNIQUE_POKESTOP = 12;
+    WITH_QUEST_CONTEXT = 13;
+    WITH_THROW_TYPE_IN_A_ROW = 14;
+    WITH_CURVE_BALL = 15;
+    WITH_BADGE_TYPE = 16;
+    WITH_PLAYER_LEVEL = 17;
+    WITH_WIN_BATTLE_STATUS = 18;
+    WITH_NEW_FRIEND = 19;
+    WITH_DAYS_IN_A_ROW = 20;
+}
+```
+
+### Conditions Detail
+When the basic fields of the condition don't cover all use-cases, reference below
+
+Some possibilities from the conditions type array were left out because they didn't seem useful to have any detail on
+
+#### Pokemon Type (type `1`)
+See [PA's type array](https://github.com/PokeAlarm/PokeAlarm/blob/master/locales/en.json#L1016) for a complete type
+id/name correlation (minus the padded zeroes)
+```json
```

Comment:
Again, I would rather see a table + a full example of what this condition looks like in json formatting